.github/workflows/update-contribs.yml

```
@@ -0,0 +1,44 @@
+name: Update Contributors
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+jobs:
+    run-meta:
+        runs-on: ubuntu-latest
+        steps:
+        - name: Check out the code
+          uses: actions/checkout@v3
+        - name: Setup Python
+          uses: actions/setup-python@v2
```

```suggestion
          uses: actions/setup-python@v4
```

Use up to date version here